sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -141,21 +142,23 @@ private[sql] class HiveSessionCatalog(
           // built-in function.
           // Hive is case insensitive.
           val functionName = funcName.unquotedString.toLowerCase(Locale.ROOT)
+          logWarning(s"Encounter a failure during looking up function:" +
```

Comment:
Nit: Encounter -> Encountered, and the first string doesn't need interpolation

Reply:
Thanks @srowen Updated!